### ISSUE TITLE
Trailing commas

### ISSUE THREAD

**Issue by Fabien-Chouteau** [AdaCore/ada-spark-rfcs#111]
### Summary

I would like to open the discussion on a "feature" called trailing comma in Python, Rust, JavaScript, Kotlin, C99, or C++11.

Anywhere there is a comma-separated list, allow a trailing comma after the last element of the list.

For example allow:
```ada
   Plop : Array_Type (1 .. 4) := (
                                  1,
                                  2,
                                  3,
                                  4,
                                  );                               
```

Since this is Ada/SPARK, we could also allow trailing semi-colon:
```ada
   procedure Plop (
                   A : Natural;
                   B : Float;
                   C : Integer;
                   );
```

### Motivation

The motivation is the same as with other languages allowing trailing commas:
- diffs that look cleaner for code reviews and version control
- easily reorder the lines
- easily comment/uncomment any item

### Caveats and alternatives

At this point I don't see a negative side for this.

As far as alternatives, I've seen some codebases where people put the comma at the beginning of the list:
```c
enum Channel {
    RED
   ,GREEN
   ,BLUE
};
```
Which works for adding new elements inside or at the end of the list. But the problem remains for the beginning.

**Comment by raph-amiard** [comment]
I agree this is a "low hanging fruit" in terms of UX. Putting that up for discussion
